scripts/us_census/acs5yr/subject_tables/common/generate_col_map.py

```diff
@@ -0,0 +1,284 @@
+"""Module to generate a column map to their corresponding statistical variable defintions."""
+import os
+import sys
+import logging
+import json
+import re
+
+import pandas as pd
+
+#TODO:Remove this import and make it from ~/data/utils/ directory when it is available
+os.system(
+    'curl https://raw.githubusercontent.com/Abilityguy/data/dcid_gen/util/statvar_dcid_generator.py -LO statvar_dcid_generator.py'
+)
+from statvar_dcid_generator import get_stat_var_dcid
+
+# intitalize the logger
+logging.basicConfig(
+    filename='column_map_generatetion_task.log',
+    format="{asctime} {processName:<12} {message} ({filename}:{lineno})",
+    style="{",
+    filemode='w')
+
+logger = logging.getLogger()
+logger.setLevel(logging.DEBUG)
+
+# regex for the definition of QuantityRanges
+# From: utils/statvar_dcid_generator.py
+_QUANTITY_RANGE_REGEX = (r"\[(?P<lower_limit>-|-?\d+(\.\d+)?) "
+                         r"(?P<upper_limit>-|-?\d+(\.\d+)?) "
+                         r"(?P<quantity>[A-Za-z]+)\]")
```

Comment:
As I commented in that PR, this can also be [USDollar 10000 190000]

Reply:
Regexes were added as experiment to disambiguate columns with the same quantity ranges pointing to different properties, this change is now dropped.